docs/content/components/subhead.md

```diff
@@ -77,14 +77,20 @@ Use the Subhead component for page headings.
 | `danger` | `Boolean` | `false` | Whether to style the heading as dangerous. |
 | `system_arguments` | `Hash` | N/A | [System arguments](/system-arguments) |
 
+:nodoc
```

Comment:
@manuelpuyol we expect all these nodoc strings?

Reply:
we do, I added that comment because of `rubocop`
those are undocumented slots, so we should work on them